### DISCUSSION TITLE
Future of Vercel Postgres

### DISCUSSION THREAD

**Discussion by fbele** [vercel/community#3384] [Feature Question] [answered]
### Question

Hello Vercel Support Team,

Since I don't really have any other option where I can post this question, I hope I can get answers to my inquiry here.
I am interested in using Vercel's feature of hosting the Postgres databases. As for now I see that this feature is still in Beta so I have a few questions about it if anybody can answer:
1. Since the feature is in Beta, what is the Roadmap for it? When will it be ready for production?
2. Is this a managed DB? How is it with backups? I couldn't find any documentation about it.
3. As for the moment, the database can be deployed only on few of the regions of Vercel's CDN. When the feature goes to production, is it to be expected, that one can choose any of the region in the CDN?

Thank you.

**Comment by leerob** [comment]
1. We're actively working on get Vercel Postgres out of beta! Stay tuned.
2. Yes, it is managed through Neon: https://vercel.com/docs/storage/vercel-postgres
3. What region are you wanting to deploy Vercel Postgres to?

**Reply by fbele** [reply]
Reply to point No.3 - It would be a great if a DB can be deployed to any region in your CDN listed here: https://vercel.com/docs/concepts/edge-network/regions
Vercel's CDN is really something what stands out compared to competition. Alternatively, there should at least also be an option to deploy the DB to more remote places too, or maybe to have an option for a server in each continent,
Personally I am interested in deploying the DB to 'cpt1' region.

**Reply by fbele** [reply]
Bump. Any update on my inquiry?

**Reply by leerob** [reply, marked as answer]
Thanks for the suggestion! We'll forward your request to Neon for adding additional region support.
